package.json

```diff
@@ -20,9 +22,11 @@
     "hyperprogress": "0.1.0",
     "hyperscript": "^1.4.7",
     "hypertabs": "^3.0.0",
+    "insert-css": "^2.0.0",
     "is-visible": "^2.0.4",
     "kvgraph": "^0.1.0",
     "map-filter-reduce": "^3.0.1",
+    "micro-css": "^0.6.2",
```

Comment:
i've only been using chrome or electron, which has `setImmediate`. i think it's a dependency of `mutant`.